drivers/virtualbox/virtualbox.go

```diff
@@ -658,8 +653,29 @@ func getAvailableTCPPort() (int, error) {
 			port = p
 			return port, nil
 		}
+		port = 0 // Throw away the port hint before trying again
 		time.Sleep(1)
 	}
+	time.Sleep(1)
```

Comment:
A most excellent question.  I suspect I may have put it in to probe if there there was a race between handing the port back to the OS, and then pouncing on with the Virtualbox NAT.  But if there is a race, a magic sleep is a lousy solution.  I'll pull it out and see if anything breaks.  In any case, bad form to not have a story for every line of the diff.